packages/auth0_fastapi/src/auth0_fastapi/auth/auth_client.py

```diff
@@ -47,37 +51,41 @@ def __init__(self, config: Auth0Config, state_store=None, transaction_store=None
                 **(config.authorization_params or {})
             },
         )
-    
-    async def start_login(self, app_state: dict = None, store_options: dict = None) -> str:
+
+    async def start_login(self, app_state: dict = None, authorization_params: dict = None, store_options: dict = None) -> str:
         """
         Initiates the interactive login process.
         Optionally, an app_state dictionary can be passed to persist additional state.
         Returns the authorization URL to redirect the user.
         """
-        options = StartInteractiveLoginOptions(app_state=app_state)
+        pushed_authorization_requests = self.config.pushed_authorization_requests
+        options = StartInteractiveLoginOptions(
+            pushed_authorization_requests=pushed_authorization_requests,
+            app_state=app_state,
+            authorization_params=authorization_params if not pushed_authorization_requests else None
+        )
```

Comment:
☝️  relevant changes on this file!
We make sure that authorization_params is forwarded only when client is not initiated for pushed_authorization_requests.